include/application/tasks/valve_demo.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * This is a toy task for actuating a valve and demonstrating
+ * the platform's valve management functionality. This task
+ * does the following:
+ * 
+ *  1. Opens the inspiratory valve.
+ *  2. Waits 5 seconds
+ *  3. Closes the inspiratory valve.
+ *  4. Opens the expiratory valve.
+ *  5. Waits 5 seconds
+ *  6. Closes the expiratory valve.
+ *  7. Opens the tank valve.
+ *  8. Waits 5 seconds
+ *  9. Closes the tank valve.
+ *  10. Repeat from step (1) 
+ * 
+ * This task should eventually be removed from the application.

```

Comment:
Please add in the comment something about a tester using this task to verify the valve actuation, with additional information on how the tester will make sure that the valves are open or close.

Reply:
@LaureMillet I will add a comment that this should be used by a tester. But I am not going to include information about how at assess the whether the valves are working or not. This is not appropriate to put in a code comment.